cmd/config/config.go

```diff
@@ -28,7 +28,9 @@ type UpfConfig struct {
 	HeartbeatInterval uint32   `mapstructure:"heartbeat_interval" json:"heartbeat_interval"`
 	HeartbeatTimeout  uint32   `mapstructure:"heartbeat_timeout" json:"heartbeat_timeout"`
 	LoggingLevel      string   `mapstructure:"logging_level" validate:"required" json:"logging_level"`
+	IPPool            string   `mapstructure:"ip_pool" json:"ip_pool"`
```

Comment:
Is it possible to use validator for IP subnet here?

Reply:
I tried using several different tags related to ip - none of them worked